docs/resources/network.md

```diff
@@ -39,17 +39,17 @@ resource "upcloud_router" "example_router" {
 
 ### Required
 
-- `ip_network` (Block List, Min: 1, Max: 1) A list of IP subnets within the network (see [below for nested schema](#nestedblock--ip_network))
```

Comment:
If I have understood right, the plugin framework does not mark any blocks required which would mark all blocks as optional in our docs 🤔

Reply:
Found one related [discussion answer](https://discuss.hashicorp.com/t/plugin-framework-blocks-dont-have-required-field/50180/2)

Reply:
Created also a PR in the hashicorp/terraform-plugin-docs repository: https://github.com/hashicorp/terraform-plugin-docs/pull/366